ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -185,6 +185,8 @@ def _get_l3_interface_cfg(self, l3_interface: dict) -> dict | None:
             "description": interface_description,
             "speed": l3_interface.get("speed"),
             "service_profile": l3_interface.get("qos_profile"),
+            "access_group_in": get(self._l3_interface_acls, f"{interface_name}.ipv4_acl_in.name"),
```

Comment:
Add some check and raise if there is no ipv4_acl_in set for wan interface.
Also add to doc.